drivers/soundwire/bus.c

```diff
@@ -967,17 +967,17 @@ int sdw_bus_prep_clk_stop(struct sdw_bus *bus)
 			simple_clk_stop = false;
 	}
 
-	if (is_slave && !simple_clk_stop) {
+	/* Don't need to inform slaves if there is no slave attached */
```

Comment:
The comment should be updated, e.g.

'there is no Slave device attached, skip remaining configurations'

Reply:
@plbossart Thanks, I just found that the same thing can be done for both `sdw_bus_prep_clk_stop` and `sdw_bus_exit_clk_stop`, so both are updated in a new push.